terraform/test/variables.tf

```diff
@@ -55,5 +59,5 @@ variable "ebs_backup_sns_topic_name" {
 }
 
 variable "ebs_backup_sns_subscription_address" {
-  default = "jasong.miller@gsa.gov"
```

Comment:
@Vermyndax Can we include multiple emails here as default SNS subscription address? Something like IDI group email and D2D group email.